x-pack/plugins/security_solution/public/common/components/filter_group/index.scss

```diff
@@ -11,7 +11,7 @@
 }
 
 .filter-group__wrapper {
-  .euiFlexGroup.controlGroup {
+  .euiPanel.controlsWrapper {
```

Comment:
Made some styling changes that require a different element to be targeted in order for the controls to be rendered the same way they were before 🙇